assets/aws/Makefile

```diff
@@ -14,7 +14,7 @@ MARKETPLACE_AMI_NAME ?=
 AWS_REGION ?= us-west-2
 
 # Teleport version
-TELEPORT_VERSION ?= 4.3.2
+TELEPORT_VERSION ?= 4.3.5
```

Comment:
Add a comment saying that this should be the last _released_ version available to the public.
Prevents someone from matching it to an unreleased value from the root `Makefile`